src/main/java/news/caughtup/model/NewsSource.java

```diff
@@ -0,0 +1,79 @@
+package news.caughtup.model;
+
+import java.util.Date;
+
+public class NewsSource {
+    private int resourceId;
+    private String name;
+    private String baseURL;
+    private String description;
+    private String rssURL;
+    private Date latestArticleDate;
```

Comment:
For this one, it's fine if we use Date. It won't be stored in the database, I changed it and it's only stored in memory. Plus, the RSS feed library returns a Date and that's the only place we need it.